test/src/test/java/org/corfudb/integration/LogReplicationLogicalGroupIT.java

```diff
@@ -141,7 +141,7 @@ public void testBasicLogicalGroupReplication() throws Exception {
         // Register client and setup initial group destinations mapping
         CorfuRuntime clientRuntime = getClientRuntime();
         LogReplicationLogicalGroupClient logicalGroupClient =
-                new LogReplicationLogicalGroupClient(clientRuntime, SAMPLE_CLIENT_NAME);
+                new LogReplicationLogicalGroupClient(clientRuntime, "00000000-0000-0000-0000-0000000000009");
```

Comment:
we can continue using the sample client name for tests.  Let us use a fixed common constant instead of hardcoding with a different value.